index.html

```diff
@@ -14,11 +14,36 @@
   <link rel="stylesheet" href="css/reset.css">
   <link rel="stylesheet" href="css/layout.css">
   <link rel="stylesheet" href="css/styles.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Proza+Libre:wght@400;500;600&display=swap" rel="stylesheet">
 </head>
 <body>
   <header>
+    <div class="container row">
```

Comment:
One element should not be both a `row` and a `container`. You should make this element the `container` and give it a child with the `row` class.